openedx/enterprise-catalog
# Config model to force-run mgmt commands

* Adds a configuration model to help force-run mgmt commands in stage/prod environments, so that
we can work around our task-locking logic in emergencies.
* Installs django-config-models and resolve version incompatibilities for some recently-released major versions.
* No longer runs Django 3 in tox validation, because django-config-models requires Django<3.

https://openedx.atlassian.net/browse/ENT-4206

Testing:
* Run `./manage.py migrate`
* Run any of the management commands e.g. `./manage.py update_content_metadata`
* Run it again to see the task lock.
* Go to http://localhost:18160/admin/catalog/catalogupdatecommandconfig/ and create a new config record, with `enabled` and `force` set to True.  Save it.
* Run the command again, see that it didn't task lock this time.
* Overwrite your config in http://localhost:18160/admin/catalog/catalogupdatecommandconfig/ with a new record to set `force` back to false.

## Post-review

Squash commits into discrete sets of changes
